doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -238,7 +238,11 @@ Non-resilient applications are sensitive to platform impairments on Compute like
 <p align="center"><b>Table 3-1:</b> Categories of applications, requirements for scheduling pods and Kubernetes features</p>
 
 Kubernetes clusters using above enhancements can implement worker nodes with "bare metal" servers (running Container Runtime in Linux host Operating System) or with virtual machines (VMs, on hypervisor).
-When running in VMs, the following requirements and configuration are needed for performance optimization: CPU Manager for vCPUs in VM. Huge pages enabled in hypervisor and mapped into VM. Hardware Topology Management with hypervisors enabled for NUMA. Node Feature Discovery and Device Plugin Framework for CPU instructions and devices exposed through hypervisor.
+When running in VMs, the following list of configurations shows what is needed for non-resilient applications:
+* CPU Manager managing vCPUs that hypervisor provides into VM.
+* Huge pages enabled in hypervisor, mapped into VM, enabled in guest OS, and mapped into pod.
+* Hardware Topology Management with NUMA enabled in hypervisor, mapped into VM, if needed enabled in guest OS, and mapped into pod.
```

Comment:
NUMA has to be enabled in the BIOS (or hardware) also.

Reply:
> NUMA has to be enabled in the BIOS (or hardware) also.

@pgoyal01 : Yes, also SMT, and both also for bare metal. I'm not sure that fits in this section. The idea of this was to compare bare metal and VMs.